configs/default/IFRC-IFLIGHT_F745_AIO.config

```diff
@@ -107,27 +118,42 @@ dma pin A03 0
 
 # feature
 feature -RX_PARALLEL_PWM
+feature -AIRMODE
```

Comment:
Why is airmode disabled here? This is a tuning setting, and not part of the hardware configuration.

Reply:
Fixed it. Thanks.